src/Symfony/Component/Security/Core/User/CurrentUserProvider.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Symfony\Component\Security\Core\User;
+
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+
+/**
+ * Current User Provider.
+ *
+ * This provider gives you the current logged in user.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class CurrentUserProvider
```

Comment:
I was not aware of that PR on SensiolabsFrameworkExtraBundle. But that is just for Controllers. This CurrentUserProvider service is meant to make it easier to get the logged in user in your other services.